.github/workflows/nonregression-eu.yml

```diff
@@ -4,16 +4,138 @@ on:
   workflow_dispatch:
   schedule:
     - cron: "0 11 * * 1-5"
-  push:
```

Comment:
@rthorn-nr why do we remove the run of the tests on push to main? Wouldn't we want to know that something pushed still passes the test?

Reply:
Never mind, I see the eu-smoke is doing the build on push to main.

Reply:
@Julien4218 my proposal is that we can run the `nonregression-eu-smoke` workflow on push to main and still run full regression testing daily at its regularly-scheduled time. This is only because the EU tests have been flaky when too many of them are running simultaneously, even with `max-parallel` set to 30.

I'm happy to delete the EU smoke test workflow and run full EU regression instead, but it will take longer to run, especially since the smoke tests are run last (just due to the directory structure -- definitely can change the script to run them first, which may speed that workflow up)